src/screens/Onboarding/StepProfile/AvatarCreatorItems.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react'
+import {ListRenderItemInfo, Pressable, View} from 'react-native'
+// Using the FlatList from RNGH allows us to nest the scroll views on Android. The default FlatList won't allow
+// scrolling inside of the vertical ScrollView
+import {FlatList} from 'react-native-gesture-handler'
+
```

Comment:
This is a cool thing to take note of. Wasn't aware that RNGH allowed for this, but it's very useful when trying to nest ScrollViews and running into problems on Android.